### ISSUE TITLE
XFA - Cannot open a pdf because of a typo in xfa/config.js

### ISSUE THREAD

**Issue by calixteman** [mozilla/pdf.js#13679]
Attach (recommended) or Link to PDF file here:
[Form_MGT-7.pdf](https://github.com/mozilla/pdf.js/files/6768253/Form_MGT-7.pdf)

When we try to open the above pdf, we get in the console:
```
Warning: XFA - an error occured during parsing and binding: TypeError: x.include is not a function
```
which is caused by a typo:
https://github.com/mozilla/pdf.js/blob/b9e84ba70ec5a4a30333b9092f316c9ec7b7f079/src/core/xfa/config.js#L174
 
So fixing the typo allows to open the pdf but there is only 1 page when we have 9 in using masterpdfeditor.

**Comment by Snuffleupagus** [comment]
I'm assuming that the `!!`-part could also be simplified, since the use of AND should already (automatically) be coercing the value to a boolean? Hence `.filter(x => x && x.includes(":"))` ought to suffice as far as I can tell.

**Comment by marco-c** [comment]
@calixteman is this fixed by #13684?

**Comment by marco-c** [comment]
Yes, it's fixed.
